setup.py

```diff
@@ -12,6 +12,7 @@
     "scrapy",
     "slack-sdk",
     "twisted>=19.7.0",
+    "itemadapter==0.7.0",
```

Comment:
In packages meant to be used as dependencies by others the dependency strategy is to allow as many versions of dependencies as possible, as to not lock users into specific versions, which can make it extremely hard to combine different packages with a shared dependency.

So here we should either go for no version at all:

 ```suggestion
    "itemadapter",
```

Or find out what minimum version of itemadapter Spidermon needs, though given the features used I think even the first version of itemadapter may work.